### PR TITLE
Add new key-rotation service to our build/deploy

### PR DESCRIPTION
**Pull request by whaught**
Issue https://github.com/google/exposure-notifications-server/issues/791

## Proposed Changes

* Include the key-rotation service for build/deploy/promote

**Release Note**

```release-note
Added key-rotation endpoint to build/deploy/promote
```

### CODE REVIEW

**Comment by mikehelmick** [comment]
there's also a test failre

**Comment by sethvargo** [comment]
Terraform looks good - lmk when you want a re-review

File: internal/integration/helpers.go [hunk 1]
```diff
@@ -186,6 +187,18 @@ func testServer(tb testing.TB) (*serverenv.ServerEnv, *http.Client) {
 	}
 	mux.Handle("/cleanup-exposure", cleanupExposureHandler)
 
+	// Key Rotation
+	keyRotationConfig := &keyrotation.Config{
```
Comment (mikehelmick) [inline internal/integration/helpers.go]: To test rotation, we we want to tune this down further? 
I'm guessing most of the time this test won't do anything.

Comment (whaught) [reply internal/integration/helpers.go]: Yes! Sorry I backed that out - I am going to make a separate PR for the test.

**Pull request merged**